src/openrct2/rct2/SeaDecrypt.cpp

```diff
@@ -0,0 +1,109 @@
+/*****************************************************************************
```

Comment:
Should we perhaps make a separate folder just for classic rather than put it in rct2. There are likely a few constants and such we will need to have.

Reply:
Not sure - the RCT2 folder isn't that full, and apart from .sea and perhaps a few constants, do we expect to be adding more?

Reply:
I didn't want to add a new folder just for one single file. If there are more constants and stuff to be added, can that be done in a new PR? @AaronVanGeffen has already updated the xcodeproject now, wouldn't want to require another fix for that.

Reply:
Leave as is